app/styles/templates/orders/_index.scss

```diff
@@ -1,5 +1,106 @@
+@import "../../_colors";
```

Comment:
This file should be imported in `app.scss` file so it'll be available for other pages. We'll need to use these colors on multiple pages.